src/igvfd/schemas/treatment.json

```diff
@@ -67,6 +70,54 @@
             ],
             "comment": "temperature must be specified if temperature_units is specified"
         },
+        "amount": {
+            "required": [
+                "depletion",
+                "amount_units"
+            ],
+            "comment": "delpetion and amount_units must be specified if amount is specified",
+            "properties": {
+                "depletion": {
+                    "const": false
+                }
+            }
+        },
+        "amount_units": {
+            "required": [
+                "depletion",
+                "amount"
+            ],
+            "comment": "depletion and amount must be specified if anount_untis is specified",
+            "properties": {
+                "depletion": {
+                    "const": false
+                }
+            }
```

Comment:
what does the "const:false" do?

Reply:
it restricts the value to false
https://json-schema.org/understanding-json-schema/reference/generic.html#constant-values